spec/Drivetrain/BatteryManagement.vspec

```diff
@@ -10,25 +10,47 @@
 # Battery management system for EV / Hybrids
 #
 
-- BatteryTemperature:
+- Battery:
+  type: branch
+  description: Battery status
+
+- Battery.Temperature:
   datatype: float
   type: sensor
   unit: celsius
   description: Temperature of the battery pack
 
-- BatteryCapacity:
-  datatype: int8
+- Battery.StateOfCharge:
+  datatype: uint8
   type: sensor
   unit: percent
   min: 0
   max: 100
-  description: Remaining capacity of the batter pack
+  description: The current charging level as a percentage of original battery performance.
```

Comment:
Seems OK for me.  State of charge would say how much the battery is charged compared to the currently usable capacity and would then always be at or near 100% after a full charge.  But there must then also be another item that states the actual amount of stored energy, so that there is some absolute value to rely upon.